app/assets/stylesheets/components/_sortable.scss

```
@@ -45,3 +45,36 @@
   margin-left: 2rem;
   align-self: center;
 }
+
+th a.sort-by {
+  display: flex;
+  align-items: center;
+  color: inherit;
+  text-decoration: none;
+}
+
+th a.sort-by.up,
+th a.sort-by.down {
+  background: govuk-colour("light-grey");
+}
+.sort-text {
+  margin-right: 6px;
+}
+.arrow {
+  width: 0;
+  height: 0;
+  border-style: solid;
+}
+.right {
+  border-width: 6px 6px 6px 6px;
+  border-color: transparent transparent transparent black;
+}
+.up {
+  border-width: 0 6px 6px 6px;
+  border-color: transparent transparent black transparent;
+ 
+}
+.down {
+  border-width: 6px 6px 0 6px;
+  border-color: black transparent transparent transparent;
+}
```

I haven't tested this so it might not work as intended, but we might be able to rewrite this as:
```suggestion
.arrow {
  width: 0;
  height: 0;
  border: 6px solid transparent;
}
.right {
  border-left-color: black;
}
.up {
  border-top-width: 0;
  border-bottom-color: black;
 
}
.down {
  border-bottom-width: 0;
  border-top-color: black;
}
```